spec/graphql/types/mutation_type_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe Types::MutationType do
+  subject { Types::MutationType }
+  # 'mutationName'        => {
```

Comment:
That was meant as a description of the layout of the hash. But if that isn't clear then it is just making it more difficult...

Reply:
maybe put and explanatory comment describing what it is above, i didn't interpret it as such but re-reading it makes some sense, any reason we have to know this as a comment instead of interpreting from the `.each` params?